sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/csv/CSVExprUtils.scala

```diff
@@ -79,4 +81,39 @@ object CSVExprUtils {
         throw new IllegalArgumentException(s"Delimiter cannot be more than one character: $str")
     }
   }
+
+  /**
+   * Helper method that converts string representation of a character sequence to actual
+   * delimiter characters. The input is processed in "chunks", and each chunk is converted
+   * by calling [[CSVExprUtils.toChar()]].  A chunk is either:
+   * <ul>
+   *   <li>a backslash followed by another character</li>
+   *   <li>a non-backslash character by itself</li>
+   * </ul>
+   * , in that order of precedence. The result of the converting all chunks is returned as
+   * a [[String]]
+   *
+   * @param str the string representing the sequence of separator characters
+   * @return a [[String]] representing the multi-character delimiter
+   * @throws IllegalArgumentException if any of the individual input chunks are illegal
+   */
+  def toDelimiterStr(str: String): String = {
+    import scala.collection.mutable.StringBuilder
+    var idx = 0
+
+    val delimiters = new StringBuilder()
```

Comment:
To clarify, this isn't allowing for multiple delimiters, but a single multi-char delimiter, right? If so I wonder if the description and docs should be updated to clarify that. This val is probably just `delimiter` right?

Reply:
Correct.  The original Jira description was clear, but I think the title was not (so I tweaked it).  When you say "docs", are you referring to a place in the source repo where I should also be looking?